generators/app/index.js

```diff
@@ -34,7 +34,13 @@ module.exports = class extends Generator {
     return this.prompt(prompts.slice(0, 2)).then(props => {
       const done = this.async();
       this.inputs = props;
-      this.stepTitle = this.inputs.appName;
+
+      if (this.inputs.appName.substr(0,4).toLowerCase() === 'step') {
```

Comment:
This could be more idiomatic and readable with a [regex](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Regular_Expressions) like `/^step-/`, which could also be used with a [`replace()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/replace) next line.

Reply:
You could also go with
`this.inputs.appName.startsWith('step-')`
Seems pretty readable